docs/ru/data_types/float.md

```diff
@@ -1,7 +1,60 @@
-Float32, Float64
-----------------
+# Float32, Float64
 
-Числа с плавающей запятой, то же, что и float, double в языке C.
-В отличие от стандартного SQL, числа с плавающей запятой поддерживают inf, -inf, а также nan-ы.
-Смотрите замечание о сортировке nan-ов в разделе "Секция ORDER BY".
-Не рекомендуется хранить числа с плавающей запятой в таблицах.
+[Числа с плавающей точкой](https://en.wikipedia.org/wiki/IEEE_754).
+
+Типы эквивалентны типам языка С:
+- `Float32` - `float`;
+- `Float64` - `double`.
+
+Из соображений производительности рекомендуем хранить данные в целочисленноми виде всегда, когда это возможно. Например, переводите в целочисленные значения числа с фиксированной точностью, такие как денежные суммы или времена загрузки страниц в миллисекундах.
+
+## Особенности использования чисел с плавающей точкой
+
+- При переводе из строки в число с плавающей точкой возможна ошибка округления.
+
+  ```sql
+  SELECT 1 - 0.9
+  ```
+
+  ```
+  ┌───────minus(1, 0.9)─┐
+  │ 0.09999999999999998 │
+  └─────────────────────┘
+  ```
```

Comment:
Example does not relate to your test, because there is no conversion from string to number.